open_duck_mini_v2/README.md

```diff
@@ -0,0 +1,19 @@
+# Open Duck Mini V2
```

Comment:
Hi @apirrone, can you enhance this README to match other models? (e.g. [here](https://github.com/google-deepmind/mujoco_menagerie/blob/main/arx_l5/README.md)). Broadly we need to know where the source of truth was derived from (if from a URDF, a commit SHA + link to the URDF would be great) and what steps were taken to create the MJCF.